components/sup/src/main.rs

```diff
@@ -232,6 +235,21 @@ fn mgrcfg_from_matches(m: &ArgMatches) -> Result<ManagerConfig> {
     if let Some(events) = m.value_of("EVENTS") {
         cfg.eventsrv_group = ServiceGroup::from_str(events).ok();
     }
+
+    if let Some(keyfile) = m.value_of("KEYFILE") {
+        let pb = PathBuf::from(keyfile);
+        if pb.exists() {
+            cfg.key_file = Some(pb);
+        }
+    }
+
+    if let Some(certfile) = m.value_of("CERTFILE") {
+        let pb = PathBuf::from(certfile);
+        if pb.exists() {
+            cfg.cert_file = Some(pb);
+        }
```

Comment:
Seems like we'd want to shut down with an error if either of these files didn't exist (or if a value was set for one but not the other), since otherwise, we'd just start up without HTTPS, and the user wouldn't know any different.

In light of that, we may even want to bundle both files into a single field in `cfg` (maybe an `Option<(PathBuf, PathBuf)>`, or something similar), since they always go together.

Reply:
Good idea